Automattic/jetpack
# Jetpack Dashboard: add new "Earn" section

## Proposed changes:

This PR adds a new "Earn" section to the Jetpack Settings at Jetpack > Settings. That section currently includes: 

- the "Ads" section, previously found at Jetpack > Settings > Traffic
- the contents of the existing `https://wordpress.com/earn/` page, but could / should be updated in the future.

**Screenshot**

- When the user is not connected:

<img width="1129" alt="image" src="https://github.com/Automattic/jetpack/assets/426388/b4a62224-c61a-4267-8413-d4e678fd9b00">

- When the user is connected:

<img width="1113" alt="image" src="https://github.com/Automattic/jetpack/assets/426388/510fc3b3-225e-414d-b523-7772646c928b">

### Other information:


## Jetpack product discussion

* p1688481163954019-slack-C0299DMPG

## Does this pull request change what data or activity we track or use?

* Yes, all clicks in the new dashboard screen are tracked, like in other areas of the dashboard.

## Testing instructions:

* Go to Jetpack > Settings > Earn
* Check contents of the new page.
    * Try this with different plans and statuses.
